### ISSUE TITLE
Please keep changelog over releases

### ISSUE THREAD

**Issue by c58** [styletron/styletron#327]
Hey there!

I'm using stryletron in production and i'm trying to keep dependencies up to date in my organization. And it seems like styletron doesn't have a nice changelog for each release so i need to spend time to figure out what changed by checking commits in order to upgrade safely.

So, please, make changelog for each release, that will save me time, and i hope not only to me but other users of the library. Maybe introduce some commits naming convention.

Thanks.

**Comment by erictaylor** [comment]
I agree. It also helps greatly when trying to identify if the TypeScript definitions need updated. So far I've had to manually peak into commits to verify there aren't any significant changes that would call for an update to the type definitions.
